pages/linux/rpmconf.md

```diff
@@ -0,0 +1,12 @@
+# rpmconf
+
+> Handle RPMNEW, RPMSAVE and RPMORIG files left over by package upgrades.
+> More information: <https://docs.fedoraproject.org/en-US/quick-docs/upgrading-fedora-offline/#sect-optional-post-upgrade-tasks>.
```

Comment:
```suggestion
> More information: <https://github.com/xsuchy/rpmconf>.
```
The README of the official GitHub repo seems a bit more complete to me, what do you think?